### PR TITLE
Fix strong name signing

### PR DESCRIPTION
**Pull request by MikhailSuendukov**
* [x] Has `CHANGELOG.md` been updated?
* [x] Are the files formatted correctly?

## Description

Fix strong name signing on building pipeline.

## Related PRs or issues

[AB#107475](https://dev.azure.com/msmobilecenter/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/107475)

### CODE REVIEW

File: .azurepipelines/build-release-packages-1es.yml [hunk 1]
```diff
@@ -200,6 +216,12 @@ extends:
           inputs:
             secureFile: 'appcenter-pair-key.snk'
 
+        - task: UseDotNet@2
```
Comment (DmitriyKirakosyan) [inline .azurepipelines/build-release-packages-1es.yml:219]: why do we need this task?

Comment (MikhailSuendukov) [reply .azurepipelines/build-release-packages-1es.yml]: I checked and removed it as unnecessary.

File: scripts/strong-named-sign.cmd [hunk 2]
```diff
@@ -13,15 +13,48 @@ setlocal
 set pathToAssemblies=%1
 set privateKey=%2
 
-call "%VSAPPIDDIR%..\Tools\VsDevCmd.bat"
+for /f "delims=" %%i in ('powershell -Command "Get-ChildItem -Path C:\ -Recurse -Directory -Filter 'Microsoft Visual Studio' -ErrorAction SilentlyContinue -Force | Select-Object -First 1 -ExpandProperty FullName"') do set vsPath=%%i
```
Comment (DmitriyKirakosyan) [inline scripts/strong-named-sign.cmd:16]: can you use vswhere?

**Review by DmitriyKirakosyan — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,10 @@
 
 ## Version 5.0.6 (Under development)
 
+### App Center
```
Comment (DmitriyKirakosyan) [inline CHANGELOG.md:5]: It's windows only.

**Review by DmitriyKirakosyan — APPROVED** [review]

**Pull request merged**